proxy/proxy.go

```diff
@@ -498,7 +501,11 @@ func (proxy *Proxy) attach(containerID string, orDie bool) error {
 		args = append(args, "--no-multicast-route")
 	}
 	if orDie {
-		args = append(args, "--or-die")
+		if killContainer {
+			args = append(args, "--or-die")
+		} else {
+			args = append(args, "--or-kill")
+		}
```

Comment:
`--or-die` is the existing behaviour; it kills the container.
The new behaviour is to kill the process.
Maybe `--or-kill-process` ?